imports.go

```diff
@@ -140,3 +172,23 @@ func isOwnPackage(path, cwd string) bool {
 	}
 	return false
 }
+
+func unique(input []string) []string {
+	u := make([]string, 0, len(input))
+	m := make(map[string]bool)
+
+	for _, val := range input {
+		if _, ok := m[val]; !ok {
+			m[val] = true
+			u = append(u, val)
+		}
+	}
+
+	return u
+}
+
+type listOutput struct {
```

Comment:
move this declaration to begin of the file, it is more obvious to find declaration of such structures when they are defined in the beginning of file